tests/unit/test_runner_manager.py

```diff
@@ -56,8 +56,8 @@ def charm_state_fixture():
             GithubRepo("test_owner", "test_repo"),
             ProxyConfig(
                 no_proxy="test_no_proxy",
-                http_proxy=TEST_PROXY_SERVER_URL,
-                https_proxy=TEST_PROXY_SERVER_URL,
+                http=TEST_PROXY_SERVER_URL,
```

Comment:
Why this change?

Reply:
It was wrong before, ProxyConfig no longer contains the `http_proxy' attrs.